rdt/transformers/datetime.py

```diff
@@ -151,6 +152,34 @@ def _fit(self, data):
                 'next_transformer': None,
             }
 
+    def _set_fitted_parameters(self, column_name, min_max_values, null_transformer, dtype='object'):
+        """Manually set the parameters on the transformer to get it into a fitted state.
+
+        Args:
+            column_name (str):
+                The name of the column for this transformer.
+            min_max_values (tuple or None):
+                None or a tuple containing the (min, max) values for the transformer.
+                Should be used to set self._min_value and self._max_value and must be
+                provided if self.enforce_min_max_values is True.
+            null_transformer (NullTransformer):
+                A fitted null transformer instance that can be used to generate
+                null values for the column.
+            dtype (str, optional):
+                The dtype to convert the reverse transformed data back to. Defaults to 'object'.
+        """
+        self.reset_randomization()
+        self.columns = [column_name]
+        self.output_columns = [column_name]
+        self._dtype = dtype
+        self.null_transformer = null_transformer
```

Comment:
We should handle and test the case where there are null values. `from_column` in NullTransformer adds a new column that should be put into `self.output_columns`.

https://github.com/sdv-dev/RDT/pull/838 for reference